auction-server/src/auction.rs

```diff
@@ -990,7 +991,49 @@ pub async fn handle_bid_svm(
         bid.transaction.message.static_account_keys(),
         submit_bid_instruction,
     )?;
-
+    verify_signatures_svm(&bid, &store.express_relay_svm.relayer.pubkey())?;
+    simulate_bid_svm(chain_store, &bid).await?;
     // TODO implement this
     Err(RestError::NotImplemented)
 }
+
+fn verify_signatures_svm(bid: &BidSvm, relayer_pubkey: &Pubkey) -> Result<(), RestError> {
+    let message_bytes = bid.transaction.message.serialize();
+    let all_signatures_valid = bid
+        .transaction
+        .signatures
+        .iter()
+        .zip(bid.transaction.message.static_account_keys().iter())
```

Comment:
are the static_account_keys always sorted as same as signatures order?

Reply:
yes, I copied this logic from the `verify_with_results` function